Cookbook-ios/FocusedTable/FocusedCollectionController.swift

```diff
@@ -0,0 +1,49 @@
+//
+//  FocusedTableController.swift
+//  Cookbook-ios
+//
+//  Created by Carolina Arcos on 3/6/20.
+//  Copyright © 2020 Wolox. All rights reserved.
+//
+
+import UIKit
+
+class FocusedCollectionController: GeneralViewController {
+    @IBOutlet weak var focusedCollectionView: UICollectionView!
+    
+    private let _colors = FocusedColors
```

Comment:
you could put this in a viewModel

Reply:
We are not using MVVM, just simple MVC because there are no big functionalities/logic

Reply:
I know, but it would be nice to have the viewmodel structure in case we want to copy the implementation